unit_tests/test_Inspector.py

```diff
@@ -59,9 +60,9 @@ def setUp(self):
         self.makeTestDB()
         self.loadData(os.path.join(dbp_testing.testsdir, 'data', 'db_dumps',
                                    'testDB_dump.json'))
-        self.inspect = imp.load_source('inspect', os.path.join(
-            dbp_testing.testsdir, 'inspector', 'rot13_L1.py'))
-
+        filename = os.path.join(dbp_testing.testsdir, 'inspector', 'rot13_L1.py')
+        self.inspect = None
```

Comment:
If you update load_source not to take the module argument, all these None assignments can go away (several in this file)